pytorch_lightning/callbacks/model_checkpoint.py

```diff
@@ -44,26 +44,33 @@ class ModelCheckpoint(Callback):
     best checkpoint file and :attr:`best_model_score` to retrieve its score.
 
     Args:
-        filepath: path to save the model file.
-            Can contain named formatting options to be auto-filled.
+        dirpath: directory to save the model file.
 
             Example::
 
                 # custom path
                 # saves a file like: my/path/epoch=0.ckpt
-                >>> checkpoint_callback = ModelCheckpoint('my/path/')
+                >>> checkpoint_callback = ModelCheckpoint(dirpath='my/path/')
+
+            By default, dirpath is `None` and will be set at runtime to the location
+            specified by :class:`~pytorch_lightning.trainer.trainer.Trainer`'s
+            :paramref:`~pytorch_lightning.trainer.trainer.Trainer.default_root_dir` or
+            :paramref:`~pytorch_lightning.trainer.trainer.Trainer.weights_save_path` arguments,
+            and if the Trainer uses a logger, the path will also contain logger name and version.
+
+        filename: model filename. Can contain named formatting options to be auto-filled.
+
+            Example::
 
                 # save any arbitrary metrics like `val_loss`, etc. in name
                 # saves a file like: my/path/epoch=2-val_loss=0.02-other_metric=0.03.ckpt
                 >>> checkpoint_callback = ModelCheckpoint(
-                ...     filepath='my/path/{epoch}-{val_loss:.2f}-{other_metric:.2f}'
+                ...     dirpath='my/path',
+                ...     filename='{epoch}-{val_loss:.2f}-{other_metric:.2f}'
                 ... )
 
-            By default, filepath is `None` and will be set at runtime to the location
-            specified by :class:`~pytorch_lightning.trainer.trainer.Trainer`'s
-            :paramref:`~pytorch_lightning.trainer.trainer.Trainer.default_root_dir` or
-            :paramref:`~pytorch_lightning.trainer.trainer.Trainer.weights_save_path` arguments,
-            and if the Trainer uses a logger, the path will also contain logger name and version.
+        filepath: path to save the model file.
+            Deprecated and will be removed in v1.3.0.
```

Comment:
Is next release 1.1.0?
```suggestion
        filepath: path to save the model file.
            
            .. warning:: .. deprecated:: 1.1.0
                Use ``dirpath`` + ``filename`` instead. Will be removed in v1.3.0.
```

Reply:
don't know.

Reply:
deprecated in 1.0 and removed in 1.2

Reply:
@ydcjeff mind check the doc failure here? Not sure what's the issue. Thanks :)